PublicAPI.md

```diff
@@ -898,6 +898,28 @@ getUserAgentString(identifier: string): string;
 const userAgentString = tns.analyticsSettingsService.getUserAgentString("tns/3.3.0");
 ```
 
+### getPlaygroundInfo
+The `getPlaygroundInfo` method allows retrieving information for projects that are exported from playground
```

Comment:
Please note when this method returns null or undefined.

Reply:
I do it here - https://github.com/NativeScript/nativescript-cli/pull/3396/files#diff-eaac680656b280b3360a73aac495df29R908. Is it ok?